src/Propel/Generator/Reverse/MysqlSchemaParser.php

```diff
@@ -264,6 +264,8 @@ public function getColumnFromRow(array $row, Table $table): Column
         // BLOBs can't have any default values in MySQL
         $default = preg_match('~blob|text~', $nativeType) ? null : $row['Default'];
 
+        $extra = $row['Extra'];
```

Comment:
Can you explain why this is necessary here? When I look at the code, it does not look like `$row` is changed between here and line 300, where you use it.

Reply:
Makes sense. Personally, I'd prefer it in line 300, as there is no benefit to moving it further up at the moment.

Reply:
Sounds good to me. I changed it accordingly.